examples/tutorials/advance_model_config/03_train_steps.ipynb

```diff
@@ -0,0 +1,446 @@
+{
```

Comment:
<p>but you use <code>tensors_to_fetch</code> as <code>fetches</code> in code and not using loss at all</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/analysiscenter/batchflow/pull/332/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>